qutebrowser/browser/downloads.py

```diff
@@ -49,6 +48,27 @@
 RetryInfo = collections.namedtuple('RetryInfo', ['request', 'manager'])
 
 
+def download_dir():
```

Comment:
Please rename this to `_download_dir()` as it's not used outside of the `downloads` module.